src/test/kotlin/com/target/liteforjdbc/postgres/PostgresDataSourceBuilderTest.kt

```diff
@@ -131,4 +134,42 @@ class PostgresDataSourceBuilderTest {
         result.password shouldBe "password"
     }
 
+    @ParameterizedTest
+    @EnumSource(TargetServerType::class)
+    fun `Test targetServerTypes`(targetServerType: TargetServerType) {
```

Comment:
i'm missing the part where it validates the `targetServerType` property.?

also, if we're testing that, do we need the other validations - seems unnecessary if those other things are already being tested. (that one's more a question for @chad-moller-target)

Reply:
@dtanner good call. I added the parameterized test but missed the assertion. Adding an assertion now and removing redundant ones.

Reply:
I have removed all but required fields and added `targetServerType` assertion.